src/bh_route_blocks.erl

```diff
@@ -99,10 +101,10 @@ block_list_to_json(Results) ->
 
 block_base_to_json({Height, Time, Hash, TxnCount}) ->
     #{
-      <<"height">> => Height,
-      <<"time">> => Time,
-      <<"hash">> => Hash,
-      <<"transaction_count">> => TxnCount
+      height => Height,
```

Comment:
Is this actually faster? Wouldn’t the atom just make it have to convert it to binary?

Reply:
but you don't create garbage on the heap, I guess I'm not sure which is better

Reply:
But that garbage is just created later when the JSON is constructed by converting the (newly constructed) atom to a binary anyway, no?

Reply:
atoms are interned, there's only ever one instance

Reply:
The conversion from atom to binary still happens right?